src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -1858,11 +1915,9 @@ public enum PowerKeepers implements LogicCardInfo {
           eff = delayed {
             before APPLY_SPECIAL_CONDITION, {
               def pcs=e.getTarget(bg)
-              if (pcs.types.contains(D)) {
-                if (ef.type == POISONED || ef.type == CONFUSED || ef.type == PARALYZED) {
-                  bc "Sidney's Stadium - [D] Pokemon can't be Asleep, Confused or Paralyzed."
-                  prevent()
-                }
+              if ( pcs.types.contains(D) and [ASLEEP, CONFUSED, PARALYZED].contains(ef.type) ) {
```

Comment:
I think `and` should be changed to `&&`

Reply:
Also the IDE says `ef.type` doesn't exist, however `ef.effectType` does exist if you want to use that

Reply:
>and
_whoops_

will correct the ef.type in a bit

Reply:
and weird on ef.type, a ton of cards seem to use it? But I'll apply the change